src/services/qrService.ts

```diff
@@ -0,0 +1,25 @@
+import QRCode from 'qrcode'
+import {NextFunction} from "express";
+import ApiError from "../exeptions/api-error";
+
+class QrService {
+    async createQrCode(id: number,
```

Comment:
you need to create type pf payload:
type QrCodeInformation = { id: number, dateTime: string, ....}